jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3816,14 +6873,20 @@ private static int binarySearchRange(char[] data, char c) {
 		return mid - (c < value ? 1 : 0);
 	}
 
-	private static char[] startCombiningAbove = { '\u0300', '\u033D', '\u0346', '\u034A', '\u0350', '\u0357', '\u0363', '\u0483', '\u0592', '\u0597',
```

Comment:
Please add a comment that this data is encoded from the Canonical_Combining_Class=Above table in https://www.unicode.org/Public/6.2.0/ucd/extracted/DerivedCombiningClass.txt

Reply:
Fixed in 45bb385